.github/workflows/continuous-integration.yaml

```diff
@@ -106,6 +106,8 @@ jobs:
             npm-${{ runner.os }}-${{ matrix.node-version }}-${{ hashFiles('package-lock.json') }}
           restore-keys: |
             npm-${{ runner.os }}-${{ matrix.node-version }}-
+      - name: Update npm
+        run: npm install --global npm
```

Comment:
Can we pin the version number, or a major version number? Is it worth conditionally doing this if the npm version is old, so the build runs faster?